### ISSUE TITLE
[Off topic] Code of http://www.seeks-project.info/site/

### ISSUE THREAD

**Issue by GreenLunar** [beniz/seeks#18]
Please create a repository with website code of http://www.seeks-project.info/site/

**Comment by GreenLunar** [comment]
BOOM!

**Comment by beniz** [comment]
hey @GreenLunar I should have a tarball somewhere if you're interested.

However, I am in the process of coupling searx with dhtfck in order to get a truely working p2p search with users sharing results etc... Alpha is a week away, patience!

This is actually the moment I want to say thank you for the long attention you have been giving the project. By spawning a new small project in the p2p + search domain I am actually willing to pay my long due tribute to people like you :)

**Comment by GreenLunar** [comment]
> hey @GreenLunar I should have a tarball somewhere if you're interested.

Certainly.  We would then put the code in https://github.com/seeksio/seeks.github.io ^_^

> However, I am in the process of coupling searx with dhtfck in order to get a truely working p2p search with users sharing results etc... Alpha is a week away, patience!

This is good news!

> This is actually the moment I want to say thank you for the long attention you have been giving the project.

You are welcome!  Thank YOU for Seeks!  This is indeed important, especially when not every Free Software project is well funded and is depended on code contributors to keep Free Software projects alive.

> By spawning a new small project in the p2p + search domain I am actually willing to pay my long due tribute to people like you :)

Thank you. I am very flattered.

**Comment by beniz** [comment]
Thanks for the kind words :) I was mentioning the new project here since the old website and doc will be deprecated. I'll try to put a tarball of the old Wordpress + Mediawiki website somewhere for you to download.

**Comment by GreenLunar** [comment]
Boom!

**Comment by GreenLunar** [comment]
Are there any news on this matter?
